summaries/2020/10086.md

```diff
@@ -2,6 +2,25 @@
 
 https://developer.apple.com/wwdc20/10086
 
-Presenters: _Example Guy, Another Person_
+Presenter: _Mark Mikin_
 
-##### TO-DO! You can contribute to this session, please see [CONTRIBUTING.md](CONTRIBUTING.md)
+- This is one of those "why" sessions. Why we should build something?
+- **Intelligent System Experience**
+  - The main idea is to be proactive
+  - Intelligence is how the OS works with the apps that people use every day to make the "every day" easier for people
+  - Intelligence is design (it should be viewed as a design practice)
+- **Living design**. What is that?
+  - The core job of the designer is to help people accomplish something
+  - One of the key was a designer enables this is is be leveraging the familiar
```

Comment:
This sentence is very confusing, I'm not sure what you mean 😅  Maybe this?

```suggestion
  - One of the key ways a designer can enable this is by leveraging the familiar
```

The "familiar" what, though? 🤔

Reply:
Oh yes. I just completely messed and missed it. This sentence is related to the first one. So idk if it can be combined and/or rephrased to show that a designer role is to help people to accomplish something by leveraging the familiar stuff that people used to.
That moment: [2:14](https://developer.apple.com/videos/play/wwdc2020/10086/)

Reply:
Got it!